crates/syn-solidity/src/yul/stmt/assignment.rs

```diff
@@ -0,0 +1,144 @@
+use crate::{Spanned, WalrusToken, YulExpr, YulFnCall, YulPath};
+
+use proc_macro2::Span;
+use std::fmt;
+use syn::{
+    parse::{Parse, ParseStream},
+    punctuated::Punctuated,
+    Result, Token,
+};
+
+/// Assignaration of one or more Yul variables with optional initial value.
+/// If multiple variables are declared, only a function call is a valid initial
+/// value.
+///
+/// Solidity Reference:
+/// <https://docs.soliditylang.org/en/latest/grammar.html#a4.SolidityParser.yulVariableAssignaration>
```

Comment:
correct link is this <https://docs.soliditylang.org/en/latest/grammar.html#a4.SolidityParser.yulAssignment>